README.md

```diff
@@ -1,13 +1,15 @@
 # Express API Project
+This is a project made during the Technigo bootcamp. The assignment was an introduction to backend, creating an API using Express. The API should have at least a couple of RESTful endpoints which return either an array of data, or a single item (from hard-coded set of data in JSON-file).
 
-Replace this readme with your own information about your project. 
+## My project
+I used a dataset about bookreviews. My API contains queries for filtering on author, title and language, and if neither is queried all books/objects will be shown. If the query doesn't return a result the API will show an empty array. I made params to find ID and ISBN, and filter on title and languages. If nothing is found an error message is shown, and for language it also lists all available languages. I also added endpoints to list a top-10 list of books with highest average rating in reviews, and to list all available authors in a sorted list without duplicates. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+There's probably a lot more combinations and useful info that one could be able to search for, but this is a start :)
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## Techs/tools used
+* Node.js
+* Express
+* JavaScript
 
 ## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://books-api-express.herokuapp.com/
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓